client/my-sites/plans/jetpack-plans/plans-filter-bar-i5/index.tsx

```diff
@@ -109,13 +110,15 @@ const PlansFilterBarI5: React.FC< FilterBarProps > = ( {
 					checked: durationChecked,
 				} ) }
 			>
-				<span className="plans-filter-bar-i5__toggle-off-label">Bill monthly</span>
+				<span className="plans-filter-bar-i5__toggle-off-label">
+					{ translate( 'Bill monthly' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Billed monthly' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/add-missing-translate-calls)